utils/getScenarioContent.tsx

```diff
@@ -56,6 +56,25 @@ export function getScenario(claimData: Claim): ScenarioType {
   }
 }
 
+/**
+ * Return whether the Claim Status should display the "continue certifying" content.
+ */
+export function continueCertifying(scenarioType: ScenarioType, claimData: Claim): boolean {
+  // If the Scenario is not scenario 5 or 6
+  // AND hasCertificationWeeksAvailable is true
+  // Then we should display the "continue certifying" content.
+  if (
```

Comment:
IMO the comment above is so clear that you could just `return ![ScenarioType ... && claimData.hasCertificationWeeksAvailable` instead of the if/else. Perfectly fine the way it is though

Reply:
This actually fails type check, so I'm going to leave it as is.